app/controllers/change_dates_controller.rb

```diff
@@ -0,0 +1,59 @@
+class ChangeDatesController < ApplicationController
```

Comment:
just calling it `DatesController`  sounds better,
but maybe there is a reason to call it `ChangeDatesController`

Reply:
I just picked what Jeremy suggested offline. I originally had DatesController

Reply:
I really don't have strong opinions

Reply:
So semantically ImportDate would make more sense